docs/constitution/pyth-dao-constitution.md

```diff
@@ -2,46 +2,45 @@
 
 ## Terminology
 
-- **Votable Token:** staked token in the governance contract
-- **Pyth Improvement Proposal (PIP)**
-- **Pyth DAO Treasury:** all tokens held in the governance smart contract that is directly governed by the Pyth DAO via on-chain voting
-- **Pyth Forum:** the system adopted by the DAO to manage PIP, organize the voting process and provide connectivity to the governance contract on the Solana Blockchain
-- **Pythian Multisig Wallet:** smart contract wallet signed by the elected members of the Pythian Council
-- **Price Feed Multisig Wallet:** smart contract wallet signed by the elected members of the Price Feed Council
-- **Operations Wallet:** wallet used by council members to submit PIPs on-chain. Each council has access to a dedicated and separate Operations Wallet. This shared wallet can be used by automated processes to make proposals or by council members to propose large proposals that would otherwise require many hardware wallet signatures.
-- **PGAS:** utility token exclusively used in the Pythnet Appchain, governed by the Pyth DAO
+- **Votable Token:** PYTH tokens staked token in the governance contract at the following address: pytGY6tWRgGinSCvRLnSv4fHfBTMoiDGiCsesmHWM6U
+- **PIP:** Pyth Improvement Proposal
+- **Pyth DAO Treasury:** all tokens held in the governance smart contract, at the following address: [treasury_wallet_address] , that is directly governed by the Pyth DAO via on-chain voting
```

Comment:
Gx4MBPb1vqZLJajZmsKLg8fGw9ErhoKsR8LeKcCKFyak for SOL treasury
9HKkxg5dpqjUEW1U2r76SpQCH7uvDMciytNYxrpwMVNc for SPL tokens